src/components/String/String.jsx

```diff
@@ -31,36 +31,32 @@ function Button({ stringNumber, note }) {
     setIsPlaying(false);
   }
 
+  function playAndStopSound() {
+    if (isPlaying) {
+      stopSound();
+    } else {
+      playSound();
+    }
+  }
+
   return (
-    <div className={styles.stringContainer}>
+    <button className={styles.stringContainer} onClick={playAndStopSound}>
```

Comment:
Add the `li` starting here, wrapping the whole of button element

Reply:
Can I return li directly from the String component instead of wrapping the whole String element?